common/membership/hashring.go

```diff
@@ -275,6 +282,42 @@ func (r *ring) ring() *hashring.HashRing {
 	return r.value.Load().(*hashring.HashRing)
 }
 
+func (r *ring) emitHashIdentifier() float64 {
+	members, err := r.peerProvider.GetMembers(r.service)
+	if err != nil {
+		r.logger.Error("Observed a problem getting peer members", tag.Error(err))
+		return -1
+	}
+	self, err := r.peerProvider.WhoAmI()
+	if err != nil {
+		r.logger.Error("Observed a problem self", tag.Error(err))
```

Comment:
I suggest changing the message to be a bit more meaningful 
Maybe "Observed a problem getting self hash ID" ?

Reply:
🤦  yes, I dunno what I was thinking